ui/src/index.js

```diff
@@ -40,6 +40,9 @@ if (basepath) {
     basename: "",
   })
 }
+browserHistory.listen(_ => {
```

Comment:
Generally for arrow functions without a first argument we use empty parens: `() => {}`